src/Microsoft.DotNet.Build.Tasks/Targets/Build.Common.targets

```diff
@@ -26,12 +26,14 @@
     Import the default package restore and resolve targets 
   
       Inputs:
+        ProjectJson - If not set defaults to $(MSBuildProjectDirectory)\project.json
         ProjectPackagesConfigFile - If not set defaults to $(MSBuildProjectDirectory)\packages.config
-        RestorePackages - If not set defaults to the existence of the file $(ProjectPackagesConfigFile)
-        ResolveNuGetPackages - If not set defaults to to the existence of the file $(ProjectPackagesConfigFile)
+        RestorePackages - If not set defaults to the existence of the file $(ProjectPackagesConfigFile) or $(ProjectJson)
+        ResolveNuGetPackages - If not set defaults to to the existence of the file $(ProjectPackagesConfigFile) or $(ProjectJson)
       
       Depends on properties:
-        NugetRestoreCommand - Used to restore the project packages
+        NugetRestoreCommand - Used to restore the project packages from packages.config
+        DnuRestoreCommand - Used to restore the project packages from project.json
```

Comment:
Is Dnu the new nuget.exe I guess I will have to wait to see that once you start consuming these changes.